conf/application.conf

```diff
@@ -19,6 +19,7 @@ application.router=prod.Routes
 application.global=uk.gov.hmrc.nisp.config.ApplicationGlobal
 
 application.session.httpOnly=true
+play.filters.headers.contentSecurityPolicy= "default-src 'self' 'unsafe-inline' localhost:9032 www.google-analytics.com data:"
```

Comment:
@roundrobin99 @andrewgee 
I've added the port for contact-frontend because the "Get Help with this page" link is an AJAX request from the browser and did not load locally without it.